### PR TITLE
Wrote changelog entry for Notebooks update

### PR DESCRIPTION
**Pull request by benji-glitsos-ga**
- [x] I have spellchecked my content using Microsoft Word, Grammarly, or otherwise.

### CODE REVIEW

**Comment by benji-glitsos-ga** [comment]
@robbibt Is there any specific info I can add to the changelog about what changes you made to DEA Tools datahandling? Maybe just one sentence long?

**Comment by robbibt** [comment]
There's some wording here: https://github.com/GeoscienceAustralia/dea-notebooks/releases/tag/0.3.1

Ideally I'd like to include a link directly to the rendered API docs for that function, but we'll need to point KH to the latest version of `stable` first.

**Comment by CEKrause** [comment]
Agree with Robbi - include a link to the actual tagged release with all the release notes. We don't want the changelog to duplicate too much content - it's just a collection point. So make sure to include the point of truth information and a link to the repo where people can find the code and the actual release.

**Comment by CEKrause** [comment]
See the notes on the changelog for the 2023-11 for how I'd suggest we style this sort of alert.

**Comment by benji-glitsos-ga** [comment]
@robbibt @CEKrause Great feedback. I've now edited this changelog entry quite a bit: https://deploy-preview-118--dea-docs.netlify.app/tech-alerts-changelog/#dea-notebooks-update

File: docs/tech-alerts-changelog/_tech_alerts_changelog.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ See the [DEA monitoring dashboard](https://monitoring.dea.ga.gov.au/).
 See [alert dated 2024-01-10](./#sentinel-2-contiguity-fix-reprocessing-commenced) for more details.
 :::
 
+## 2024-02-20: DEA Notebooks update
+
+A minor update to the DEA Tools package that includes updates to tide modelling and parallel processing functions. It updates the `datahandling` module. For more detail, see the [dea-notebooks 0.3.1 release notes](https://github.com/GeoscienceAustralia/dea-notebooks/releases/tag/0.3.1).
```
Comment (robbibt) [inline docs/tech-alerts-changelog/_tech_alerts_changelog.md:22]:
```suggestion
A minor update to the DEA Tools package that includes updates to tide modelling and parallel processing functions. It also adds a new `load_reproject` function to the `datahandling` module that allows reading and reprojecting external raster data. For more detail, see the [dea-notebooks 0.3.1 release notes](https://github.com/GeoscienceAustralia/dea-notebooks/releases/tag/0.3.1).
```

Comment (benji-glitsos-ga) [reply docs/tech-alerts-changelog/_tech_alerts_changelog.md]: Thanks. I added your suggestion.

**Review by robbibt — APPROVED** [review]
Looks great!

**Pull request merged**